src/modules/common/components/Page.js

```diff
@@ -31,14 +31,14 @@ type PropType = {|
   content: string,
   theme: ThemeType,
   navigation: NavigationScreenProp<*>,
-  navigateToIntegreatUrl: NavigateToIntegreatUrlParamsType => void,
-  navigateToFeedback: (positive: boolean) => void,
+  navigateToIntegreatUrl?: NavigateToIntegreatUrlParamsType => void,
+  navigateToFeedback?: (positive: boolean) => void,
```

Comment:
Would be nice to have these two implemented for the disclaimer as well. (Also the TFunction should not be nullable I guess?)

Reply:
I'll create a task, it's also missing for the sprungbrett and wohnen extra.
Regarding the nullable TFunction, since it's only used/required for the feedback, I wrapped the FeedbackBox inside a `translate` instead of passing it here.